public/api/v2/config

```diff
@@ -23,9 +23,9 @@
     "saml": "aufSAMLn.de"
   },
   "specialLinks": {
-    "privacy": "test",
-    "termsOfUse": "test",
-    "imprint": "test"
+    "privacy": "https://example.com/privacy",
+    "termsOfUse": "https://example.com/termsOfUse",
+    "imprint": "https://example.com/imprint"
```

Comment:
This could be removed again, but then again it's not really necessary since it's mock data anyway